src/client/scripts/memberHeader.js

```diff
@@ -144,4 +143,17 @@ const validation = (function(){
         areWeLoggedIn,
     })
 
+})();
+
+const pihedron = (function () {
```

Comment:
"Pihedron" doesn't explain what this does. Can you just move its logic into a function called, maybe setActiveHeaderLink, or something, and put that function inside the above IIFE?